homeassistant/components/switchbot/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "switchbot",
   "name": "SwitchBot",
   "documentation": "https://www.home-assistant.io/integrations/switchbot",
-  "requirements": ["PySwitchbot==0.34.1"],
+  "requirements": ["PySwitchbot==0.35.0"],
```

Comment:
I think the library needs to handle possible exceptions from the `requests.post` call now, as callers won't expect that or catch a broad exception anymore.

https://github.com/Danielhiversen/pySwitchbot/blob/271b1ad8b2f5ace98ee50614ff3c0e44fdc109af/switchbot/devices/lock.py#L120

Reply:
d4e82b3f785adf2c200986fddde492f975a43ea3